portfolio/src/main/webapp/style.css

```diff
@@ -24,4 +24,8 @@ footer {
   padding: 3px;
   background-color: DarkSalmon;
   color: white;
+}
+
+#center-alignment {
```

Comment:
This should be named #main-title or something similar.

It's better to name CSS classes and IDs based on what they refer to on the page instead of what they style something as (e.g. prefer .footnote to .small-text).